github_cog.py

```diff
@@ -0,0 +1,111 @@
+        else:
+            g = Github(studybot.TOKENS.get('GITHUB_API_KEY'))
+            # Get the user object.
+            try:
+                g_user = g.get_user(github_name)
+            except:
+                await ctx.message.reply('The GitHub user you requested cannot be found. Double-Check your spelling.')
+                return()
+            # Get the organization object.
+            try:
+                g_org = g.get_organization(studybot.github_org_name)
+            except:
+                msg = 'The GitHub organization cannot be found. Please try again later, '
+                msg += 'or contact a mod if the issue persists.'
+                await ctx.message.reply(msg)
+                return()
+
+            # Verify that the user has not already invited a GitHub account to the org.
+            with closing(studybot.db_conn.cursor()) as conn:
+                async with studybot.lock:
+                    query = 'SELECT github_id FROM github_invites WHERE discord_id = ?'
+                    conn.execute(query, (ctx.author.id, ))
+                    value = conn.fetchone()
+                    if value is None:
+                        # Verify that the user is not already in the org.
+                        if g_org.has_in_members(g_user):
+                            await ctx.message.reply('That user is already in the organization, and cannot be invited.')
+                            return()
+                        # Invite the user to the organization.
+                        else:
+                            g_org.invite_user(g_user)
+
+                        # Add the Discord ID and the GitHub ID to the database table
+                        query = 'INSERT INTO github_invites (discord_id, github_id) VALUES (?, ?)'
+                        conn.execute(query, (ctx.author.id, g_user.id))
+
+                        # Send a confirmation message in Discord.
+                        await ctx.message.reply(f'Invited {g_user.login} to the GitHub Organization.')
+                    else:
+                        msg = f'Cannot invite {g_user.login} to the organization, because you have already invited '
+                        msg += 'another GitHub account. Please contact a moderator if this is a mistake.'
+                        await ctx.message.reply(msg)
+                    studybot.db_conn.commit()
+
+    @commands.command(name='github-reset', help=studybot.help_dict.get('github-reset'))
+    @commands.has_role(studybot.admin_role_id)
+    async def reset_user(self, ctx, discord_user: discord.User = None):
+        if not discord_user:
+            await ctx.message.reply('You need to tag the user to reset their abilities.')
+            return()
+
+        # Find who the user previously invited to the org.
+        with closing(studybot.db_conn.cursor()) as conn:
+            async with studybot.lock:
+                query = 'SELECT github_id FROM github_invites WHERE discord_id = ?'
```

Comment:
You are running the same query more than once. Maybe it a function in this class/util function?

Reply:
That's a good point, I didn't notice.